implementations/rust/ockam/ockam_api/README.md

```diff
@@ -11,6 +11,123 @@ and trustfully with cloud services and other devices.
 This crate supports the creation of a fully-featured Ockam Node
 (see [`NodeManager`](https://github.com/build-trust/ockam/blob/2fc6d7714a4e54f8734c172ad6480fedc6e3629c/implementations/rust/ockam/ockam_api/src/nodes/service.rs#L87) in [`src/nodes/service.rs`](https://github.com/build-trust/ockam/blob/2fc6d7714a4e54f8734c172ad6480fedc6e3629c/implementations/rust/ockam/ockam_api/src/nodes/service.rs)).
 
+## Configuration
+
+A `NodeManager` maintains its configuration as a list of directories and files stored under
+the `OCKAM_HOME` directory (`~/.ockam`) by default:
+```shell
+root
+├─ credentials
+│  ├─ c1.json
+│  ├─ c2.json
+│  └─ ...
+├─ defaults
+│  ├── credential -> ...
+│  ├── identity -> ...
+│  ├── node -> ...
+│  └── vault -> ...
+├─ identities
+│  ├─ data
+│  │  ├─ authenticated-storage.lmdb
+│  │  └─ authenticated-storage.lmdb-lock
+│  ├─ identity1.json
+│  ├─ identity2.json
+│  └─ ...
+├─ nodes
+│  ├─ node1
+│  │  ├─ default_identity -> ...
+│  │  ├─ default_vault -> ...
+│  │  ├─ policies-storage.lmdb
+│  │  ├─ policies-storage.lmdb-lock
+│  │  ├─ setup.json
+│  │  ├─ stderr.log
+│  │  ├─ stdout.log
+│  │  └─ version
+│  ├─ node2
+│  └─ ...
+├─ projects
+│  └─ default.json
+├─ trust_contexts
+│  └─ default.json
+└─ vaults
+   ├─ vault1.json
+   ├─ vault2.json
+   ├─ ...
+   └─ data
+      ├─ vault1.lmdb
+      ├─ vault1.lmdb-lock
+      ├─ vault2.lmdb
+      ├─ vault2.lmdb-lock
+      └─ ...
+```
+## `credentials`
+
+Each file stored under the `credentials` directory contains the credential for a given identity.
+Those files are created with the `ockam credential store` command. They are then read during the creation of
+a secure channel to send the credentials to the other party
+
+## `defaults`
+
+This directory contains symlinks to other files or directories in order to specify which node,
+identity, credential or vault must be considered as a default when running a command expecting those
+inputs
+
+## `identities`
+
+This directory contains one file per identity and a data directory. An identity file is created
+with the `ockam identity create` command or created by default for some commands (in that case the
```

Comment:
Would identities created by the identity service be also stored here? If not are they stored anywhere?